openshift/topics/before_you_begin.adoc

```diff
@@ -18,7 +18,7 @@ The {project_openshift_product_name} image version number between 7.0 and 7.3 ar
 ====
 
 === Initial Setup
-The Tutorials in this guide follow on from and assume an OpenShift instance similar to that created by performing https://docs.openshift.com/container-platform/latest/install/index.html[the installation of the OpenShift Container Platform cluster].
+The Tutorials in this guide follow on from and assume an OpenShift instance similar to that created by performing https://docs.openshift.com/container-platform/3.11/install/running_install.html[the installation of the OpenShift Container Platform cluster].
```

Comment:
Latest Red Hat Single Sign-On v7.3.7 for OpenShift image is known to work properly on latest available OCP v4.3.5.X (including templates). So there's no need to point to outdated OCP v3.1 cluster installation IMHO.

I would replace the suggested `https://docs.openshift.com/container-platform/3.11/install/running_install.html` link with some general purpose OCP v4.x one (for example the `https://docs.openshift.com/container-platform/latest/architecture/architecture-installation.html` one), which seems to provide high-level overview of available platforms & the installation process itself (so the user can choose specific installation procedure depending on the platform chosen: AWS, Openstack, bare metal etc.)

@hmlnarik @pdrozd What's your take on this -^ ?